yt_dlp/extractor/spreaker.py

```diff
@@ -58,15 +60,18 @@ def duration(key):
 
 
 class SpreakerIE(InfoExtractor):
-    _VALID_URL = r'''(?x)
-                    https?://
-                        api\.spreaker\.com/
-                        (?:
-                            (?:download/)?episode|
-                            v2/episodes
-                        )/
-                        (?P<id>\d+)
-                    '''
+    _VALID_URL = [
+        r'''(?x)
+        https?://
+            api\.spreaker\.com/
+            (?:
+                (?:download/)?episode|
+                v2/episodes
+            )/
+            (?P<id>\d+)
+        ''',
```

Comment:
since the diff already changes all these lines, we might as well do this, which imo is more readable if we're doing multiple patterns
```suggestion
        r'https?://api\.spreaker\.com/(?:(?:download/)?episode|v2/episodes)/(?P<id>\d+)',
```